cli/tests/data/help/package.txt

```diff
@@ -1,93 +0,0 @@
-                          <package-name>
-    dcos package install [--cli | [--app --app-id=<app-id>]]
-                         [--package-version=<package-version>]
-                         [--options=<file>]
-                         [--yes]
-                         <package-name>
-    dcos package list [--json --app-id=<app-id> --cli <package-name>]
-    dcos package search [--json <query>]
-    dcos package repo add [--index=<index>] <repo-name> <repo-url>
-    dcos package repo remove <repo-name>
-    dcos package repo list [--json]
-    dcos package uninstall [--cli | [--app --app-id=<app-id> --all]]
-                           <package-name>
-    dcos package update
-
-Commands:
-    describe
-        Get specific details for packages.
-    install
-        Install a package.
-    list
-        Print a list of the installed DC/OS packages.
-    search
-        Search the package repository.
-    repo add
-        Add a package repository to DC/OS.
-    repo remove
-        Remove a package repository from DC/OS.
-    repo list
-        Print the package repository sources. Possible sources include a local
-        file, HTTPS, and Git.
-    uninstall
-        Uninstall a package.
-    update
-        This command has been deprecated. Repositories are dynamically updated
-        as they are added by `dcos package repo add`.
-
-Options:
-    --all
-        All packages.
-    --app
-        Application only.
-    --app-id=<app-id>
-        The application ID.
-    --cli
-        Command line only.
-    --config
-        Print the the configurable properties for the `marathon.json` file.
-    --index=<index>
-        The numerical position in the package repository list. Package
-        repositories are searched in descending order. By default the Universe
-        repository first in the list.
-    -h, --help
-        Print usage.
-    --info
-        Print a short description of this subcommand.
-    --options=<file>
-        Path to a JSON file that contains customized package installation options.
-    --package-version=<package-version>
-        The package version to install.
-    --package-versions
-        Print all versions for this package.
-    --render
-        Collate the marathon.json package template with the values from
-        config.json and --options. If not provided, print the raw templates.
-    --version
-        Print version information.
-    --yes
-        Turn off interactive mode and assume "yes" is the answer to all prompts.
-
-Positional Arguments:
-    <package-name>
-        Name of the DC/OS package in the package repository.
-    <query>
-        Pattern to use for searching the package repository.  You can use
-        complete or partial values.
-    <repo-name>
-        Name of the package repository. For example, `Universe`.
-    <repo-url>
-        URL of the package repository. For example,
-        https://universe.mesosphere.com/repo.
```

Comment:
Redundant file. After moving docopt spec to a text file we should just read that file in our tests.